pkg/compact/compact.go

```diff
@@ -63,9 +63,6 @@ type syncerMetrics struct {
 	garbageCollectionDuration prometheus.Histogram
 	compactions               *prometheus.CounterVec
 	compactionFailures        *prometheus.CounterVec
-	indexCacheBlocks          prometheus.Counter
```

Comment:
No more metrics?

Reply:
They were never there. I'm not sure what this is supposed to even show. Either way, no one has missed this so I have removed it 😅